buildSrc/subprojects/cleanup/src/main/kotlin/org/gradle/gradlebuild/testing/integrationtests/cleanup/TestFilesCleanUpPlugin.kt

```diff
@@ -29,11 +29,5 @@ open class TestFilesCleanUpPlugin : Plugin<Project> {
     override fun apply(project: Project): Unit = project.run {
         val testFilesCleanup = extensions.create<TestFileCleanUpExtension>("testFilesCleanup", objects)
```

Comment:
Maybe this extension should now be added by the `CiReportingPlugin` - or the extension should provide a way to query for the tmp test files and have a method doing the cleanup.

WDYT?

Reply:
> Maybe this extension should now be added by the CiReportingPlugin

I'll prefer this way.

Reply:
Then let's at least add a note where the extension is consumed.